CHANGELOG.md

```diff
@@ -138,3 +152,16 @@ Other minor additions
   *-inverseˡ : ∀ p → 1/ p * p ≡ 1ℚ
   *-inverseʳ : ∀ p → p * 1/ p ≡ 1ℚ
   ```
+
+* Added new proofs in `Algebra.Properties.CommutativeSemigroup.Divisibility`:
+  ```agda
+  x∣y∧z∣x/y⇒xz∣y : ∀ {x y z} → ((x/y , _) : x ∣ y) → z ∣ x/y → x ∙ z ∣ y
+  x∣y⇒zx∣zy      : ∀ {x y} z → x ∣ y → z ∙ x ∣ z ∙ y
+  ```
+
+* Added new proofs in `Algebra.Properties.CancellativeCommutativeSemiring`:
+  ```agda
+  x*y≈0⇒x≈0⊎y≈0 : Decidable _≈_ → ∀ {x y} → x * y ≈ 0# → x ≈ 0# ⊎ y ≈ 0#
+  x≉0∧y≉0⇒xy≉0  : Decidable _≈_ → ∀ {x y} → x ≉ 0# → y ≉ 0# → x * y ≉ 0#
+  xy∣x⇒y∣1      : ∀ {x y} → x ≉ 0# → x * y ∣ x → y ∣ 1#
```

Comment:
```suggestion
  x*y≈0⇒x≈0⊎y≈0 : Decidable _≈_ → x * y ≈ 0# → x ≈ 0# ⊎ y ≈ 0#
  x≉0∧y≉0⇒xy≉0  : Decidable _≈_ → x ≉ 0# → y ≉ 0# → x * y ≉ 0#
  xy∣x⇒y∣1      : x ≉ 0# → x * y ∣ x → y ∣ 1#
```

Reply:
Changed as suggested, with removing `*` from names and replacing `⊎` with disjunction in names.